hi-ml-azure/testazure/testazure/test_himl.py

```diff
@@ -771,6 +769,8 @@ def _mock_sweep(*args: Any, **kwargs: Any) -> MagicMock:
 
     dummy_root_directory = tmp_path
     dummy_entry_script = dummy_root_directory / "my_entry_script"
+    dummy_experiment_name = "my_experiment"
+    dummy_experiment_name = himl.effective_experiment_name(dummy_experiment_name, dummy_entry_script)
```

Comment:
I don't get why this is needed?

Reply:
This is as the environment variables used for experiment name creation are set per branch in the GH runner, which causes this test to fail due to me changing when `effective_experiment_name()` is called during the v2 job submission.